libp2p/protocols/pubsub/gossipsub/behavior.nim

```diff
@@ -233,12 +234,15 @@ proc handleIWant*(g: GossipSub,
         trace "peer sent iwant", peer, messageID = mid
         let msg = g.mcache.get(mid)
         if msg.isSome:
+          libp2p_gossipsub_mcache_hit.observe(1)
```

Comment:
Isn't this the same as the number of IWANT sent?
I guess not if we have multiple request in a single IWANT

Reply:
IWANT received would be, but it's not the same. I'm exposing both on the committed dashboard, and numbers are surprisingly different. So much that there might be some scaling issue as well.

Reply:
Oh, we are in the handleIWant, I thought we were in the handeIHave
Having very different numbers would indicate that a peer sending a IWant generally missed a lot of messages

But if the ratio is bigger than 15 (iWantBudget iirc), it's a bug sumewhere